scss/bootstrap/bootstrap.scss

```diff
@@ -1,19 +1,17 @@
-@import "vendor/text-angular/text-angular";
-@import "vendor/angular-xeditable/angular-xeditable";
-@import "vendor/font-awesome/font-awesome";
-@import "vendor/perfect-scrollbar/main";
-@import "vendor/ui-bootstrap/ui-bootstrap";
-@import "vendor/ui-select/ui-select";
-@import "vendor/select2/select2-bootstrap";
+@import '../angular/vendor/*';
```

Comment:
use `SCSSROOT`

Reply:
Unfortunately `SCSSROOT` does not support wildcards like `/*`.

Reply:
see https://github.com/compucorp/civihr-employee-portal-theme/pull/325#discussion_r174040994